ui/app/AppLayouts/Chat/panels/qmldir

```diff
@@ -1 +1,2 @@
 SuggestionBoxPanel 1.0 SuggestionBoxPanel.qml
+ChatTimePanel 1.0 ChatTimePanel.qml
```

Comment:
Where's the actual file move? :)

Reply:
It's duplicated; it's also under "imports/shared/panels/chat"

Reply:
Hm, I just added it to qmldir to use it outside chat module. But having it in shared make sense, i remove it from chat